python/pyramid/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  website: trypyramid.com
```

Comment:
```suggestion
  website: https://trypyramid.com/
```

Reply:
This URL will be displayed in results (I dynamically adds `https://`) so we do not need this `here`